exercises-toolbox/8-all/corona/loesung.py

```diff
@@ -13,12 +13,15 @@
 spalten = data.dtype.names
 
 plt.bar(data['Datum'], data['Zuwachs_positiver_Testergebnisse_zum_Vortag'], label='Positive Fälle')
-liste = []
+liste = np.array([])
```

Comment:
please don't call a numpy array liste, call it what it contains. So something like `n_positive_tests`